src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int sideOne;
+    private int sideTwo;
+
+    public RightTriangle(String color, int sideOne, int sideTwo) {
+        super(color);
+        this.sideOne = sideOne;
+        this.sideTwo = sideTwo;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: triangle, area: " + getArea() + " sq. units, "
+                + "sideOne: " + sideOne + "units, " + "sideTwo: " + sideTwo + "units, "
+                + "color:" + getColor());
+    }
+
+    @Override
+    public double getArea() {
```

Comment:
According to the checklist, `draw()` and `getArea()` methods are conceptually different from each other and should not be in the same interface. Consider creating separate interfaces for them.